### PR TITLE
github: Add issue template

### LINKED ISSUE #62: Add github issue template
Copy https://github.com/kata-containers/runtime/blob/master/.github/ISSUE_TEMPLATE.md into this repo.

### PR DESCRIPTION
**Pull request by jodh-intel**
Add a template to guide the user on raising github issues.

Fixes #62.

Signed-off-by: James O. D. Hunt <james.o.hunt@intel.com>

### CODE REVIEW

**Comment by jodh-intel** [comment]
Hi @klynnrif - ptal.

**Comment by jodh-intel** [comment]
Hi @klynnrif - ptal.

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+(replace this text with an explanation of what you thought would happen)
+
+# Actual result
+
+(replace this text with details of what actually happened)
+
+---
+
+(replace this text with the output of the `kata-collect-data.sh` script, after
+you have reviewed its content to ensure it does not contain any private
```
Comment (klynnrif) [inline .github/ISSUE_TEMPLATE.md]: you review its content to ensure it does not contain any private

Comment (jodh-intel) [reply .github/ISSUE_TEMPLATE.md]: Hi @klynnrif - we could change this, but please note that we're using standard wording for all the repos and we already have this template in the following places:

- https://github.com/kata-containers/runtime/blob/master/.github/ISSUE_TEMPLATE.md
- https://github.com/kata-containers/agent/blob/master/.github/ISSUE_TEMPLATE.md

And the (still) pending:

- https://github.com/kata-containers/proxy/pull/63

Comment (klynnrif) [reply .github/ISSUE_TEMPLATE.md]: Ah! Gotcha :) Okay - I'll change my review. Thanks

**Pull request merged**